tree.go

```diff
@@ -48,6 +48,15 @@ type Rect struct {
 	Height int64 `json:"height"`
 }
 
+// WindowProperties correspond to X11 window properties
```

Comment:
Could you add a link to https://build.i3wm.org/docs/ipc.html#_tree_reply here please? (Like in the comment on Node.)

Reply:
Comment updated :)